qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java

```diff
@@ -23,7 +23,7 @@
 	private static final By GIVEN_NAME = By.cssSelector("#adminui-givenName-field");
 	private static final By GENDER_FEMALE = By.cssSelector("#adminui-gender-1-field");
 	private static final By ADD_USER_ACCOUNT = By.id("adminui-addUserAccount");
-	private static final By USER_NAME = By.cssSelector("#adminui-username-field");
+	private static final By USERNAME_FIELD = By.cssSelector("#adminui-username-field");
```

Comment:
How do you generate different username here.

Reply:
The param passed in [setUsername()](https://github.com/openmrs/openmrs-contrib-qaframework/blob/5bb1758a8737c188e79e14f2250fe554b042026d/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java#L77) during its call is a randomly generated string.